src/ralph/admin/sitetrees.py

```diff
@@ -21,9 +21,15 @@
 
 activate(settings.LANGUAGE_CODE)
 
+
+def ralph_item(*args, **kwargs):
+    kwargs['access_loggedin'] = True
```

Comment:
maybe use `setdefault` here (is `access_loggedin` is defined, we don't want to overwrite it)